hack/jenkins/minishift_linux_integration_tests_kvm.sh

```diff
@@ -0,0 +1,230 @@
+#!/bin/bash
+
+# Copyright 2017 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# This script runs the integration tests on a Linux machine for the KVM Driver
+
+# The script expects the following env variables:
+# MINIKUBE_LOCATION: GIT_COMMIT from upstream build.
+# COMMIT: Actual commit ID from upstream build
+# EXTRA_BUILD_ARGS (optional): Extra args to be passed into the minikube integrations tests
+# access_token: The Github API access token. Injected by the Jenkins credential provider.
+
+set -e
+
+OS_ARCH="linux-amd64"
+VM_DRIVER="kvm"
+JOB_NAME="Minishift-Linux-KVM"
+BINARY=out/bin/minishift
+ISO=file://$(pwd)/out/minikube-testing.iso
+EXTRA_FLAGS="--show-libmachine-logs"
+
+# Download files and set permissions
+mkdir -p out/
+
+curl -L http://artifacts.ci.centos.org/minishift/minishift/master/latest/linux-amd64/minishift out/minishift
+gsutil cp gs://minikube-builds/${MINIKUBE_LOCATION}/docker-machine-driver-* out/
+gsutil cp gs://minikube-builds/${MINIKUBE_LOCATION}/minikube-testing.iso out/
```

Comment:
The minikube iso isn't guaranteed to be uploaded on every PR.  It's only uploaded on the PRs that have an ISO change and require a rebuild.  If this doesn't exist, you can 

1. just exit 0
2. test with the default iso image.  We don't expose this in a great way but you can parse either the makefile or the minikube start helptext for the default version.  The right way is probably for us to upload the latest iso image at `gs://minikube/releases/latest/minikube.iso`, which we currently don't do.

Reply:
@r2d4 I would go with option 2. how hard will it be to upload latest iso to ISO_BUCKET, I can have a look later today if that takes not much time.